x/stakeibc/keeper/unbonding_records_get_host_zone_unbondings_msgs_test.go

```diff
@@ -395,38 +395,57 @@ func (s *KeeperTestSuite) TestGetBalanceRatio() {
 	testCases := []struct {
 		unbondCapacity keeper.ValidatorUnbondCapacity
 		expectedRatio  sdk.Dec
+		errorExpected  bool
 	}{
 		{
 			unbondCapacity: keeper.ValidatorUnbondCapacity{
 				BalancedDelegation: sdkmath.NewInt(0),
 				CurrentDelegation:  sdkmath.NewInt(100),
 			},
 			expectedRatio: sdk.ZeroDec(),
+			errorExpected: false,
```

Comment:
Yes but the check is if tc.ErrorExpected... so you are saying the default when initializing bool in a struct is to be false? otherwise we need it... can we keep it just to be very clear?

Reply:
yeah the default is false. Not a big deal either way, so it's fine if you want to leave it to be explicit, but the typical testing pattern for these go tests is to exclude the field in the non-error case, and include it in the error case. (The most common pattern is generally to either use an error type or a string for the error.)

And I think in this case, if you include either errorExpected _or_ expectedRatio, the tests read nicely:
```go
{
    unbondCapacity: X,
    expectedRatio: A
},
{
    unbondCapacity: Y,
    expectedRatio: B
},
{
    unbondCapacity: Z,
    errorExpected: true
},
```